documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -196,6 +196,64 @@ Following is an example of a message:
 }
 ----
 
+[[vitess-ordered-transaction-metadata]]
+=== Ordered Transaction Metadata
+You can configure {prodname} to include additional metadata in data change event records.
+Such supplemental metadata can assist downstream consumers in processing messages in the correct order  when repartitioning, or some other disruption, might otherwise lead to data being consumed out of order.
```

Comment:
```suggestion
Such supplemental metadata can assist downstream consumers in processing messages in the correct order when repartitioning, or some other disruption, might otherwise lead to data being consumed out of order.
```